proof/crefine/X64/ADT_C.thy

```diff
@@ -541,6 +541,10 @@ end
 
 context state_rel begin
 
+definition
+  "ioapic_nirqs_from_to_H cstate \<equiv>
+     \<lambda>x. if x \<le> of_nat maxNumIOAPIC then ioapic_nirqs_' cstate.[unat x] else 0"
```

Comment:
Is the 'from' needed in this name, and if so should it say what it's from?

Reply:
That is totally a typo, thanks for spotting it!